CHANGELOG.asciidoc

```diff
@@ -38,6 +38,10 @@ Notes:
 [float]
 ===== Features
 
+* feat: Add `captureAttributes` boolean option to `apm.captureError()` to
+  allow *disabling* the automatic capture of Error object properties. This
+  is useful for cases where those properties should not be sent to the APM
+  Server, e.g. for performance (large string fields) or security (PII data).
 * feat: Add `log_level` central config support. {pull}1908[#1908] +
```

Comment:
Do we think it's worth mentioning the signature change to `parseError` in the changelog?  Or better to leave it out since this is, technically speaking, a private API that folks shouldn't be calling directly, or should be ready to accept the risks of relying on a private(-ish?) API?  (I'm split down the middle on this personally -- no compelling clear case for either approach)

Reply:
Unless I missed something, it is a completely private API (the whole lib/parsers.js is private) -- as private as you can get with JS without going to lengths -- so I'm not inclined to update the changelog.